neuromation/cli/job.py

```diff
@@ -1059,3 +1074,27 @@ def calc_statuses(status: Sequence[str], all: bool) -> Set[JobStatus]:
             statuses = defaults
 
     return set(JobStatus(s) for s in statuses)
+
+
+@asynccontextmanager
+async def port_forward_simle(
+    root: Root, job_id: str, local_port: int, job_port: int, no_key_check: bool
+) -> AsyncIterator[None]:
+    async with root.client.jobs.port_forward(
+        job_id, local_port, job_port, no_key_check=no_key_check
+    ):
+        yield
+
+
+@asynccontextmanager
+async def port_forward_reconnect(
+    root: Root, job_id: str, local_port: int, job_port: int, no_key_check: bool
+) -> AsyncIterator[None]:
+    while True:
+        try:
+            async with root.client.jobs.port_forward(
+                job_id, local_port, job_port, no_key_check=no_key_check
+            ):
+                yield
+        except (ConnectionError, ClientConnectionError):
+            click.echo(f"Reconnect localhost:{local_port} => {job_id}:{job_port}")
```

Comment:
feature idea: let's print time when the reconnect happened? otherwise the user will have a long list of identical messages.